Packs/TwitterIOCHunter-FullDailyFeed/Integrations/TwitterIOCHunterFullDailyFeed/TwitterIOCHunterFullDailyFeed.py

```diff
@@ -29,42 +33,60 @@ def find_type_and_value(indicatordata):
         return 'Error', ''
 
 
-def get_indicators_command(client):
+def get_indicators_command(client, url, feed_tags=None, tlp_color=None):
     listofindicators = []
-    result = client._http_request('GET', '/')
+    result = client._http_request('GET', full_url=url)
     for item in result:
         typeofindicator, valueofindicator = find_type_and_value(item)
         for newitem in valueofindicator:
             data = {'type': typeofindicator,
                     'value': newitem,
                     'service': 'Twitter IOC Hunter',
-                    'rawJSON': {'reference': item.get('reference'), 'tweet': item.get('tweet')},
+                    'fields': {
+                        'firstseenbysource': item.get('tweet').get('timestamp'),
+                        'tags': feed_tags,
+                        'reportedby': item.get('tweet').get('user')},
+                    'rawJSON': item,
                     'score': 3
                     }
+        if tlp_color:
+            data['fields']['trafficlightprotocol'] = tlp_color
         listofindicators.append(data)
     return listofindicators
 
 
 def main():
+    type_of_feed = demisto.params().get('typeoffeed')
     base_url = 'http://www.tweettioc.com/v1/tweets/daily/full'
+    user_url = 'http://www.tweettioc.com/v1/tweets/daily/full/user/'
+    tags_url = 'http://www.tweettioc.com/v1/tweets/daily/ioc/hashtags/'
+    feed_tags = demisto.params().get('feedTags')
+    tlp_color = demisto.params().get('tlp_color')
+    filter_to_use = demisto.params().get('filtertouse')
     verify_certificate = not demisto.params().get('insecure', False)
     proxy = demisto.params().get('proxy', False)
     demisto.info(f'Command being called is {demisto.command()}')
+    if type_of_feed == 'Username':
+        url = f'{user_url}{filter_to_use}'
+    elif type_of_feed == 'Hashtag':
+        url = f'{tags_url}{filter_to_use}'
+    else:
+        url = base_url
     try:
         client = BaseClient(
-            base_url=base_url,
+            base_url=url,
             verify=verify_certificate,
             proxy=proxy)
         if demisto.command() == 'test-module':
             # This is the call made when pressing the integration Test button.
-            result = test_module(client)
+            result = test_module(url)
```

Comment:
If you change the test_module to use client._http_request, send client here instead

Reply:
yeah, will change that.